app/views/layouts/_footer.html.erb

```diff
@@ -1,5 +1,5 @@
-<div class="container empty-div">
-</div>
+<!-- <div class="container empty-div">
+</div> -->
```

Comment:
remove if not needed

Reply:
Added it till we make the footer sticky, then it can be removed.